fml/memory/weak_ptr.h

```diff
@@ -84,6 +84,14 @@ class WeakPtr {
     return *this ? ptr_ : nullptr;
   }
 
+  T* getUnsafe() const {
```

Comment:
Add a TODO and file a bug to remove this, documenting that this is basically a means to lock down what we can while opting into the current behaviour until we've eliminated the last two unsafe derefs. Specifically note that no new uses are permitted; this is just a stopgap until we've eliminated the last two.